includes/Traits/Types.php

```diff
@@ -154,6 +155,24 @@ public function get_allowed_image_mime_types() : array {
 		return array_values( array_intersect( $allowed_mime_type, $image_mime_type ) );
 	}
 
+	/**
+	 * Returns a list of audio mime types.
+	 *
+	 * Used for the allowlist when adding background audio.
+	 *
+	 * @since 1.10.0
+	 *
+	 * @return array List of allowed mime types.
+	 */
+	public function get_allowed_audio_mime_types(): array {
+		return [
+			'audio/mpeg',
+			'audio/aac',
+			'audio/wav',
+			'audio/ogg',
+		];
+	}
```

Comment:
```suggestion
	public function get_allowed_audio_mime_types(): array {
		$allowed_mime_types = [
			'audio/mpeg',
			'audio/aac',
			'audio/wav',
			'audio/ogg',
		];

		return array_values( array_intersect( $allowed_mime_types, get_allowed_mime_types() ) );
	}
```

Reply:
Ensure that this list is filtered by `get_allowed_mime_types`, so the user can not upload an mp3, that this is filtered out.